src/askscript/parser/askscript.grammar.pegjs

```diff
@@ -270,6 +270,7 @@ mapEntryList =
 
 mapEntry = 
     i:identifier ws* ':' ws* v:value {                 return new ask.MapEntry(i, v) }
+    / ws* i:string ws* ':' ws* v:value {                 return new ask.MapEntry(i, v) }
```

Comment:
Isn't the first `ws*` superfluous? Otherwise it would also be needed in the first rule.  Also, let's align "return new" to each other.

Reply:
Well, it does not break for me:
![image](https://user-images.githubusercontent.com/12752520/96281029-0c15b080-0fd9-11eb-92a5-7b79b7ae0b1f.png)

![image](https://user-images.githubusercontent.com/12752520/96281049-146deb80-0fd9-11eb-8ff5-5822b255bf07.png)

and it actually shouldn't break, because `ws*` are consumed in mapEntryList, the only place where mapEntry is used:
![image](https://user-images.githubusercontent.com/12752520/96281142-37000480-0fd9-11eb-9838-70af59753242.png)

Reply:
That is strange - it used to fail for me...
Maybe there was some caching issue with the grammer build locally.
Removed the `ws*`